### DISCUSSION TITLE
All checks have failed: we cannot make changes to the indicator

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1516] [Q&A]
we tried to make changes to the value of indicator 5.4.2 for 2020. 
When we created Pull request: All checks have failed. 
Previously, these changes were made successfully.

**Comment by christoph-LSN** [comment]
@ukraine-sdg having a lok at your actions I found: 

`yaml.scanner.ScannerError: mapping values are not allowed here
  in "meta/15-3-1.yml", line 3, column 152`

[https://github.com/ukraine-sdg/data/runs/4712116421?check_suite_focus=true#step:5:51](https://github.com/ukraine-sdg/data/runs/4712116421?check_suite_focus=true#step:5:51) 

not sure but may be try to remove the ":" or put it in " " in this file: https://github.com/ukraine-sdg/data/blob/339999c062748f4fc3fb2b7b1861de482c9b9e4c/meta/15-3-1.yml#L3

**Comment by LucyGwilliamAdmin** [comment]
@ukraine-sdg has Christoph's response resolved this for you?
